modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/views/stories/StoryEpilogueView.kt

```diff
@@ -52,7 +59,7 @@ fun StoryEpilogueView(
 
         Spacer(modifier = modifier.weight(1f))
 
-        HeartImage()
+        PulsatingHeart()
```

Comment:
Nice!

Reply:
The only thing I think it needs matt is the last story if you try to skip the story it just loops the last story it should close the story 🤔 😉

(other then a story top 15-25 podcasts the current story should say add the time for the each of the top 5😉, theirs allways next year tho),
 (last story should also collect all the story's and allows sharing of the 9 as a group as thats better for social engagement )

Reply:
Your feedback is valuable, @CookieyedCodes! Thanks for sharing it. 🙏  I'll add it to the backlog.